athena-jdbc/src/main/java/com/amazonaws/athena/connectors/jdbc/manager/JdbcSplitQueryBuilder.java

```diff
@@ -367,4 +388,156 @@ protected String appendLimitOffset(Split split, Constraints constraints)
     {
         return " LIMIT " + constraints.getLimit();
     }
+
+    protected String appendLimitOffsetWithValue(String limit, String offset)
+    {
+        return "LIMIT " + limit;
+    }
+
+    protected SqlDialect getSqlDialect()
```

Comment:
I think this should be abstract, and we require specific connectors to implement which SQL dialect they support